.github/workflows/android-codestyle.yml

```
@@ -0,0 +1,28 @@
+name: Java CodeStyle
```

Probably should be:
```suggestion
name: Android CodeStyle
```

Or change the name of the file (since this does all JAVA files?)